webpack/assets/javascripts/compute_resource/ovirt.js

```diff
@@ -57,7 +57,7 @@ export function instanceTypeSelected(item) {
       url,
       data: `instance_type_id=${instanceType}`,
       success(result) {
-        $('[id$=_memory]').val(result.memory);
+        $('[id$=_memory]').val(result.memory / 1024 / 1024).trigger('change');
```

Comment:
instead of triggering `change` we should init all spinners after the ajax completes in `reloadOnAjaxComplete`: 
```javascript
tfm.numFields.initAll()
```

Reply:
doesnt it make more sense to agree on one unit for all compute resources? then we can have a consistent api and drive alignment in rails vs js?